src/content/upgrades/merge/eth-issuance/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: How The Merge impacts ether supply
+description: Breakdown on how The Merge will impact ether supply
+lang: en
+sidebar: true
+---
+
+# How The Merge impacts ether supply {#how-the-merge-impacts-ether-supply}
+
+<Card
+emoji=":chart_decreasing:">
+
+- Mining rewards ~13,000 ETH/day pre-merge
+- Staking rewards ~1,600 ETH/day pre-merge
+- **After The Merge, only the ~1,600 ETH per day will remain, dropping total new ETH issuance by ~90%**
+- <Emoji text=":fire:" size="1" /> The burn: At an average gas price of at least 16 gwei, at least 1,600 ETH is burned every day, which effectively brings net ETH inflation to zero or less post-merge.
```

Comment:
I'm fine with keeping the emoji in the heading below but think it makes this harder to read without adding much. Not strongly opposed to keeping it if you disagree.